.eslintignore

```diff
@@ -1,2 +1,3 @@
 __spec__.js
 __definition__.js
+*.md
```

Comment:
ESlint is currently incorrectly parsing `*.md` files resulting in parsing errors